Development/DCM/DCMDatabase/dbpm.py

```diff
@@ -5,8 +5,7 @@
 
 from src.dcm_constants           import *
 from peewee                      import *
-
-from DCMUserAccountManager.duam  import * 
+import DCMUserAccountManager.duam  
```

Comment:
why did you change this from the existing format?

Reply:
Still attempting to get the class construction on line 62 to work, which use the class User from DCMUserAccountManager.duam

Reply:
those two lines are equivalent though?